app/src/main/java/ch/epfl/favo/user/User.java

```diff
@@ -2,15 +2,19 @@
 
 import java.time.LocalDate;
 
+import ch.epfl.favo.common.DatabaseWrapper;
+
 /**
  * This class contains all the relevant information about users TODO: It should implement parcelable
  * so that it can be injected in views
  */
 public class User {
 
+  private String id;
   private String name;
   private String email;
   private String deviceId;
+  private String notificationId;
   private LocalDate birthDate;
   private int activeAcceptingFavors;
   private int activeRequestingFavors;
```

Comment:
We also need a field for the Location of the user

Reply:
I guess this is also part of the next PR

Reply:
ok let me add that